pipeline_dp/dp_computations.py

```diff
@@ -347,7 +353,10 @@ def compute_dp_mean(count: int, sum: float, dp_params: MeanVarParams):
         dp_params.max_contributions_per_partition,
         dp_params.noise_kind,
     )
-    return dp_count, dp_mean * dp_count, dp_mean
+
+    # Clamps dp_mean to be in the range [low, high].
```

Comment:
Atm PipelineDP doesn't do any postprocessing on DP results. The idea is that in case of API users would like to do statistical analysis, postprocessing would make things harder. 

We can introduce some standard postprocessing (like clipping to high, low or rounding), but there should be way to turn off it and we probably would like to introduce post-processing layer, separate from core DP layer. So I'd suggest not to clip mean for now and then to think about post-processing. WDYT?

Reply:
Separating the post-processing from the core DP layer makes sense. We can do this at the dp_engine layer (in `aggregate()`) perhaps.

Removing this for now.